doc/src/conf.py

```diff
@@ -372,7 +372,13 @@
 \usepackage{letltxmacro}
 \LetLtxMacro\OldLaTeX\LaTeX
 \AtBeginDocument{\DeclareRobustCommand{\LaTeX}{\text{\OldLaTeX}}}
-'''
+\let\OldUnderscore\_
+\makeatletter
+\AtBeginDocument{\sbox\sphinxcontinuationbox{\spx@opt@verbatimcontinued}}
```

Comment:
this `\AtBeginDocument` is probably not needed but I think LaTeX does font related things there, so I am delaying to that location.